nubia/internal/plugin_interface.py

```diff
@@ -136,3 +136,10 @@ def getBlacklistPlugin(self):
         Any return other then 0 will block command execution
         """
         return CommandBlacklist()
+
+    def get_prompt_tokens(self):
+        """
+        Override this and return your own prompt for interactive mode.
+        Expected to return a list of pygments Token tuples
+        """
+        return None
```

Comment:
It would be better to just return the default `[Token.Username, getpass.getuser()]` list here (compared to setting it in `interactive.py`).